dev/halyard_install_component.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
shoudl this be in spinnaker/install as opposed to spinnaker/dev?
The intent was that runtime scripts are in install, where developer only scripts (not part of an operational deployment) are in dev.

Otherwise LGTM